### PR TITLE
Bump version to v0.5.3

### PR DESCRIPTION
**Pull request by charliermarsh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## 0.5.3
+
+### Enhancements
+
+- Allow conflicting extras in explicit index assignments ([#9160](https://github.com/astral-sh/uv/pull/9160))
```
Comment (konstin) [inline CHANGELOG.md:7]: This is a huge thing, do we want to highlight it and link the docs page?

**Review by konstin — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (charliermarsh) [reply CHANGELOG.md:7]: That's a good idea. I'll add a blurb to the top.

**Review by BurntSushi — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,84 @@
 # Changelog
 
+## 0.5.3
+
+This release includes support for conflicting optional dependencies and dependency groups in the uv resolver, including
+the ability to specify dependency sources (like index assignment) on a per-extra or per-group basis.
```
[the remark this answers is not in the document]
Comment (charliermarsh) [reply CHANGELOG.md:6]: Yo...

[the remark this answers is not in the document]
Comment (zanieb) [reply CHANGELOG.md]: Sorry :D

**Review by zanieb — APPROVED** [review]

**Pull request merged**